pkg/node/config_defaults.go

```diff
@@ -36,6 +37,46 @@ var DefaultRequesterConfig = RequesterConfigParams{
 	MinBacalhauVersion: model.BuildVersionInfo{
 		Major: "0", Minor: "3", GitVersion: "v0.3.26",
 	},
+
+	EvalBrokerVisibilityTimeout:    60 * time.Second,
+	EvalBrokerInitialRetryDelay:    1 * time.Second,
+	EvalBrokerSubsequentRetryDelay: 30 * time.Second,
+	EvalBrokerMaxRetryCount:        10,
+
+	WorkerCount:                  runtime.NumCPU(),
+	WorkerEvalDequeueTimeout:     5 * time.Second,
+	WorkerEvalDequeueBaseBackoff: 1 * time.Second,
+	WorkerEvalDequeueMaxBackoff:  30 * time.Second,
+}
```

Comment:
more motivation to work on https://github.com/bacalhau-project/bacalhau/issues/2614 😅